analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs

```diff
@@ -32,6 +32,14 @@ public class SonarLintXmlReader
 
     private readonly SonarLintXml sonarLintXml;
 
+    public string[] Exclusions { get; private set; }
+    public string[] Inclusions { get; private set; }
+    public string[] GlobalExclusions { get; private set; }
+    public string[] TestExclusions { get; private set; }
+    public string[] TestInclusions { get; private set; }
+    public string[] GlobalTestExclusions { get; private set; }
+    public List<SonarLintXmlRule> ParametrizedRules { get; private set; }
```

Comment:
There's a code smell for this but I prefer to ignore it.
I read in the docs that List is a good choice when enumerating data more than once and that's the case here:
https://github.com/SonarSource/sonar-dotnet/blob/a60ea62ae62df3a6854ced9e33849436a4f5b7c0/analyzers/src/SonarAnalyzer.Common/Helpers/ParameterLoader.cs#L54